data/create-tables.js

```diff
@@ -32,12 +32,13 @@ async function run() {
                     zipcode INTEGER NOT NULL
                 );           
                 CREATE TABLE favorites (
-                    id VARCHAR(512) NOT NULL,
-                    favorite VARCHAR(512) NOT NULL,
+                    id SERIAL PRIMARY KEY,
+                    favorite_id VARCHAR(512) NOT NULL,
```

Comment:
Could we do breed_id instead? I think that makes it a bit clearer that it's referring to the breed rather than being it's own id

Reply:
that makes sense.